app/server/src/static-site-builder/args.ts

```diff
@@ -0,0 +1,15 @@
+const doc = `
+Usage:
+  builder <path-to-config> <dest-path> <path-to-mustache-views>
```

Comment:
it can change between development and production, it depends on the folder structure of the dist folder, which may not be the same as the folder structure of our app/server directory, i guess we can also force them to be the same and hardcode that path in wodin builder as well but felt nice to give that flexibility

Reply:
I'd assumed this would always be done from a development context, but yeah I suppose it doesn't have to be!